<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* =========================
   MAIN WINDOW
   ========================= */
QWidget {
    background-color: #191919;
    color: white;
    font-family: Segoe UI;
    font-size: 13px;
}

/* =========================
   LOGIN CARD (ORANGE BORDER)
   ========================= */
QFrame {
    background-color: transparent;
    padding: 16px;
}

/* =========================
   TEXT
   ========================= */
QLabel {
    color: #E0E0E0;
}

QLabel#lblTitle {
    color: #FF9800;
    font-size: 24px;
    font-weight: bold;
}

/* =========================
   INPUT FIELDS (BORDER STYLE)
   ========================= */
QLineEdit {
    background-color: transparent;
    color: white;
    border: 1.5px solid #FF9800;
    border-radius: 8px;
    padding: 4px;
}

QGroupBox {
    background-color: transparent;
    color: white;
    border: 1.5px solid #FF9800;
    border-radius: 8px;
    padding: 12px;
}

QComboBox {
    background-color: transparent;
    color: white;
    border: 1.5px solid #FF9800;
    border-radius: 8px;
    padding: 4px;
}

QDateEdit {
    background-color: transparent;
    color: white;
    border: 1.5px solid #FF9800;
    border-radius: 8px;
    padding: 4px;
}

QLineEdit:focus {
    border: 2px solid #FFA726;
}

/* =========================
   LOGIN BUTTON (OUTLINE STYLE)
   ========================= */
QPushButton {
    background-color: transparent;
    color: #FF9800;
    font-weight: bold;
    border: 2px solid #FF9800;
    border-radius: 10px;
    padding: 10px;
}

QPushButton:hover {
    background-color: rgba(255, 152, 0, 25);
}

QPushButton:pressed {
    background-color: rgba(255, 152, 0, 60);
    color: black;
}

/* =========================
   DISABLED BUTTON
   ========================= */
QPushButton:disabled {
    border-color: #555;
    color: #555;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Searching</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="radioButton1">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Search by Invoice No</string>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton2">
          <property name="text">
           <string>Search by RegistrationDate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton3">
          <property name="text">
           <string>Search by LoginDate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton4">
          <property name="text">
           <string>Search by Explanation</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>90</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="pgInvoiceNo">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLineEdit" name="leExplanation">
             <property name="placeholderText">
              <string>Type a word in Explanation</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btnSearchInvoiceNo">
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pgRegstrationDate">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QDateEdit" name="deRegstrationDate">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSearchLoginDate_2">
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pgExplanation">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QDateEdit" name="dateEdit_3">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit_2">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btnSearchExplanation">
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pgLoginDate">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="inputMask">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Invoice No</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSearchLoginDate">
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableView {
    gridline-color: #ccc;
    font-size: 14px;
}

QHeaderView::section {
    background-color: #2c3e50;
    color: white;
    padding: 4px;
    border: none;
}
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="lineWidth">
        <number>4</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveasexcel">
       <property name="text">
        <string>Save as excel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
